packages/dev/parcel-transformer-mdx-docs/MDXTransformer.js

```diff
@@ -54,6 +54,8 @@ module.exports = new Transformer({
 
             exampleCode.push(code);
 
+            node.value = node.value.replace(/\n*\/\/\/- begin collapse -\/\/\/(.|\n)*\/\/\/- end collapse -\/\/\//g, '').trim();
```

Comment:
what does this do?

Reply:
😄 it removes code from being included in the docs between those two comments, e.g. if it's repeated from a previous example. kinda hacky, but it works. I'll add a comment.